### PR TITLE
Added Phantom Theives Logo

### PR DESCRIPTION
**Pull request by Xephorix**

### CODE REVIEW

File: web/_js/atlas.js [hunk 1]
```diff
@@ -517,7 +517,8 @@ var atlas = [
...
-{"id": 470, "name": "Gem Galaxies", "description": "Gem Galaxies is a game on ROBLOX inspired by Steven Universe that is dedicated to roleplaying: the symbol that was created is of their diamond insignia.", "website": "https://www.roblox.com/games/273326777/Gem-Galaxies", "subreddit": "", "center": [ 1255.5, 897.5 ], "path": [ [ 1255.5, 891.5 ], [ 1250.5, 898.5 ], [ 1251.5, 899.5 ], [ 1255.5, 900.5 ], [ 1261.5, 899.5 ], [ 1256.5, 892.5 ] ] }
+{"id": 470, "name": "Gem Galaxies", "description": "Gem Galaxies is a game on ROBLOX inspired by Steven Universe that is dedicated to roleplaying: the symbol that was created is of their diamond insignia.", "website": "https://www.roblox.com/games/273326777/Gem-Galaxies", "subreddit": "", "center": [ [ 1255.5, 897.5 ], "path": [ 1255.5, 891.5 ], [ 1250.5, 898.5 ], [ 1251.5, 899.5 ], [ 1255.5, 900.5 ], [ 1261.5, 899.5 ], [ 1256.5, 892.5 ] ] }
```
[the remark this answers is not in the document]
Comment (Tais993) [reply web/_js/atlas.js:520]: Missing comma results in the map not loading

**Review by Tais993 — CHANGES_REQUESTED** [review]

File: web/_js/atlas.js [hunk 2]
```diff
@@ -517,7 +517,8 @@ var atlas = [
...
-{"id": 470, "name": "Gem Galaxies", "description": "Gem Galaxies is a game on ROBLOX inspired by Steven Universe that is dedicated to roleplaying: the symbol that was created is of their diamond insignia.", "website": "https://www.roblox.com/games/273326777/Gem-Galaxies", "subreddit": "", "center": [ 1255.5, 897.5 ], "path": [ [ 1255.5, 891.5 ], [ 1250.5, 898.5 ], [ 1251.5, 899.5 ], [ 1255.5, 900.5 ], [ 1261.5, 899.5 ], [ 1256.5, 892.5 ] ] }
+{"id": 470, "name": "Gem Galaxies", "description": "Gem Galaxies is a game on ROBLOX inspired by Steven Universe that is dedicated to roleplaying: the symbol that was created is of their diamond insignia.", "website": "https://www.roblox.com/games/273326777/Gem-Galaxies", "subreddit": "", "center": [ [ 1255.5, 897.5 ], "path": [ 1255.5, 891.5 ], [ 1250.5, 898.5 ], [ 1251.5, 899.5 ], [ 1255.5, 900.5 ], [ 1261.5, 899.5 ], [ 1256.5, 892.5 ] ] },
```
Comment (Tais993) [inline web/_js/atlas.js:520]:
```suggestion
{"id": 470, "name": "Gem Galaxies", "description": "Gem Galaxies is a game on ROBLOX inspired by Steven Universe that is dedicated to roleplaying: the symbol that was created is of their diamond insignia.", "website": "https://www.roblox.com/games/273326777/Gem-Galaxies", "subreddit": "", "center":  [ 1255.5, 897.5 ], "path": [ [ 1255.5, 891.5 ], [ 1250.5, 898.5 ], [ 1251.5, 899.5 ], [ 1255.5, 900.5 ], [ 1261.5, 899.5 ], [ 1256.5, 892.5 ] ] },
```
If you didn't notice, you modified Gem Galaxies which resulted in your PR not working.
You can view the preview [here](https://deploy-preview-335--place-atlas.netlify.app/), but yeah, it's loading infinity.
After this it's hopefully fixed? Sorry for the many reviews

**Review by Tais993 — CHANGES_REQUESTED** [review]

File: web/_js/atlas.js (hunk 2 above)
Comment (Xephorix) [reply web/_js/atlas.js]: No problem, thanks for letting me know!

**Comment by Tais993** [comment]
Yes it works now, you can try it out [here](https://deploy-preview-335--place-atlas.netlify.app/)!

**Pull request closed without merging**